.github/actions/publish/publish-wasm/action.yml

```diff
@@ -18,6 +18,10 @@ runs:
         override: true
         target: wasm32-unknown-unknown
 
+    # Download a pre-compiled wasm-bindgen binary.
+    - name: Install wasm-bindgen-cli
+      uses: jetli/wasm-bindgen-action@v0.1.0
```

Comment:
for other more niche actions we pin to commit instead of version, I think we should do the same here

Reply:
Pinned to a commit hash.